.circleci/config.yml

```diff
@@ -46,9 +46,9 @@ jobs:
             npm version $(node -e "const currentVersion=require('./package.json').version; const firstTwoDots=currentVersion.substring(0, currentVersion.lastIndexOf('.')+1); console.log(firstTwoDots);")$CIRCLE_BUILD_NUM
       - run:
           name: Authenticate with registry
-          command: curl -u $ARTIFACTORY_PUBLISHING_USER:$ARTIFACTORY_PUBLISHING_PASSWORD https://pillarproject.jfrog.io/pillarproject/api/npm/auth >> ~/common-models/.npmrc
+          command: echo "//registry.npmjs.org/:_authToken=$PILLAR_NPM_TOKEN" > ~/common-models/.npmrc
```

Comment:
@vnovacev can we add the PILLAR_NPM_TOKEN to the project in circleci?

Reply:
sure, will do it right now

Reply:
@antonioned  - Is that something @vnovacev will do on circleci or you want me to do?

Reply:
`PILLAR_NPM_TOKEN` env var is added